constants.py

```diff
@@ -1,2 +1,7 @@
-STATUS_SUFFIX = "CUDSTATUS"
-SEVERITY_SUFFIX = "CUDSEVR"
\ No newline at end of file
+from csv import DictReader
+from typing import Dict, List
+
+CSV_FAULTS: List[Dict] = []
+for row in DictReader(open("faults.csv")):
+    if row["PV Suffix"]:
+        CSV_FAULTS.append(row)
```

Comment:
Is there potentially a different name we would want to give this file? constants.py implies that it has a bunch of constants, but this is just holding the list of faults from our csv. I hesitate to think of a better name though since we already have one or two other files with "fault" in their name

Reply:
lol yeah it's a legacy. It started out as a ton of constants but I kept getting import errors so I kept having to move stuff out until only this was left